Reference hsd dependency by github link, not npm package

With shakedex in Bob Wallet, the app (Bob wallet) fails to start because `names.json` is missing in `hsd`'s published package.

To use shakedex with Bob, it should reference the `hsd`'s GitHub repo directly, like how Bob Wallet does now: https://github.com/kyokan/bob-wallet/blob/master/package.json#L150

If the file doesn't exist, the app doesn't open:
```
Module not found: Error: Can't resolve './names.json' in '/media/data/Projects/handshake/bob-wallet/node_modules/shakedex/node_modules/hsd/lib/covenants'
```

--- a/package.json
+++ b/package.json
@@ -27,7 +27,7 @@
     "commander": "7.0.0",
     "date-fns": "2.17.0",
     "hs-client": "0.0.9",
-    "hsd": "^2.3.0",
+    "hsd": "https://github.com/handshake-org/hsd/tarball/a94ce87",
     "inquirer": "7.3.3",
     "level": "6.0.1",
     "node-fetch": "2.6.1",

